tasks/section_4/cis_4.1.2.x.yml

```
@@ -233,7 +233,7 @@
 - name: "AUTOMATED | 4.1.11 | PATCH | Ensure use of privileged commands is collected"
   block:
   - name: "AUTOMATED | 4.1.11 | AUDIT | Ensure use of privileged commands is collected"
-    shell: for i in  $(df | grep '^/dev' | awk '{ print $NF }'); do find $i -ignore_readdir_race -xdev -type f -perm -4000 -o -type f -perm -2000; done
+    shell: for i in  $(df | grep '^/dev' | awk '{ print $NF }'); do find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null; done
```

I moved this fix I previously made into a custom override.  This is just restoring it to how upstream has it.

---

```suggestion
    shell: for i in  $(df | grep '^/dev' | awk '{ print $NF }'); do find $i -x -type f -perm -4000 -o -type f -perm -2000 2>/dev/null; done
```
Not that you have to change this. but i think that xdev is now depricated. just fyi.